examples/dynamic-bridge-app/linux/README.md

```diff
@@ -1,14 +1,14 @@
-# CHIP Linux Bridge Example
+# Matter Linux Dynamic Endpoints Bridge Example
 
 An example demonstrating a simple lighting bridge and the use of dynamic
 endpoints. The document will describe the theory of operation and how to build
-and run CHIP Linux Bridge Example on Raspberry Pi. This doc is tested on
+and run Matter Linux Bridge Example on Raspberry Pi. This doc is tested on
 **Ubuntu for Raspberry Pi Server 20.04 LTS (aarch64)** and **Ubuntu for
 Raspberry Pi Desktop 20.10 (aarch64)**
 
 <hr>
 
--   [CHIP Linux Bridge Example](#chip-linux-bridge-example)
+-   [Matter Linux Bridge Example](#matter-linux-bridge-example)
```

Comment:
I think the name here should match the top-level heading of the file. Now, it does not and may be that's why CI is failing with the error:

``` 
/home/runner/work/connectedhomeip/connectedhomeip/matter/docs/_build/src/examples/dynamic-bridge-app/linux/README.md:11: WARNING: 'myst' cross-reference target not found: 'matter-linux-bridge-example' [myst.xref_missing]
```